.storybook/main.js

```diff
@@ -4,7 +4,13 @@ const storybookDefaultConfig = require( '@automattic/calypso-storybook' );
 
 const storybookConfig = storybookDefaultConfig( {
 	stories: [
-		'../client/**/*.stories.{js,jsx,ts,tsx}',
+		'../client/components/**/*.stories.{js,jsx,ts,tsx}',
+
+		// TODO: Excluding these paths for now as they are currently broken and
+		// also add an extreme amount of time to the Storybook startup.
+		// '../client/my-sites/**/*.stories.{js,jsx,ts,tsx}',
+		// '../client/sites/**/*.stories.{js,jsx,ts,tsx}',
```

Comment:
I vote for deleting those broken stories altogether. 

I don't see much value in having these stories anyway, even if they were working.

We can also remove the `ReduxDecorator` and any other unused supplementing utils.

Reply:
Confirmed with @taipeicoder that we can delete them.

Done in 75d8c10c69f443c8654b47bc6c9ccd2dbf57b099